app/assets/javascripts/instruments.js

```diff
@@ -1,8 +1,8 @@
 $(document).ready(function() {
   new FullCalendarConfig($("#calendar"), {
     header: {
-      left: '',
-      center: 'title',
+      left: 'title',
```

Comment:
This title is referring to the date range, right? Why are we moving this to the left? Historically it's been centered, if you look at NU prod for example...